client/coral-embed-stream/src/containers/Stream.js

```diff
@@ -113,6 +113,7 @@ class StreamContainer extends React.Component {
   componentDidMount() {
     if (this.props.previousTab) {
       this.props.data.refetch();
+      this.props.setCommentCountCache(0);
```

Comment:
I need advice on this.

Here's the issue: Let's say you have ignored a user with a several recent comments. Refresh. The stream loads, and EmbedQuery and the comment counts queries aren't factoring in the comments from the ignored user.

Now go to the 'My profile' tab. Stop ignoring that user. Now switch back to the 'Comments' tab, which triggers the 'refetch()' above this line. Now asset.comments will include the comments by the previously ignored user. *However*, nothing has reset the commentCountCache, while asset.commentCount has gone up (by the amount of comments by the previously ignored user). This causes NewCount to think there are new comments, and thus render.

So... is this the best place to reset the comment count cache? Since EmbedQuery (asset.comments) and commentCountCache are so tightly coupled, is there any way we can make it so that whenever EmbedQuery is re-fetched, we also reset the commentCountCache?